wizarr/umbrel-app.yml

```
@@ -0,0 +1,34 @@
+manifestVersion: 1
+id: wizarr
+name: Wizarr
+tagline: An advanced invitation and management system for media servers
+category: media
+version: "2025.10.4"
+port: 5690
+description: >-
+  🧙 Wizarr is an application created to streamline the process of managing and onboarding users for media servers. It provides a centralized system that allows administrators to generate invitation links, which can be shared with new users to grant them access to the server. This process eliminates the need for manual setup and reduces administrative overhead, making it easier to manage multiple users efficiently. The application ensures that every new user receives guidance through the setup process, helping them configure their clients and connect to the server in a straightforward manner.
+
+
+  The app also offers a system for tracking invitations and user activity, giving administrators detailed insights into who has joined, when they joined, and how they interact with the server. This tracking allows for better management of access permissions, including options for temporary memberships or restricted access levels. By monitoring activity and requests, Wizarr ensures that the server remains organized and that users receive the appropriate level of access and support.
+
+
+  In addition, Wizarr integrates with various platforms and communication tools to automate certain tasks, such as sending notifications or coordinating user onboarding. It supports a guided onboarding experience, providing clear instructions and steps to ensure that users can navigate the server environment without confusion. This reduces support requests and makes the user experience smoother and more intuitive.
+
+
+  Overall, Wizarr is focused on making user management seamless, efficient, and transparent. It provides administrators with the tools to monitor interactions, handle requests, and maintain control over access, while giving users an easy and structured way to join and use the server. By combining invitation management, activity tracking, and automated guidance, Wizarr enhances the overall server experience for both administrators and users.
+developer: Wizarr
+website: https://wizarr.org/
```

I think there’s some confusion between the original project and the one mentioned on the `website` field. You’ve linked the website for a different project [https://github.com/wizarrrrr/wizarr](https://github.com/wizarrrrr/wizarr) while the repository and Docker image in the `docker-compose.yml` refer to [https://github.com/wizarrrr/wizarr](https://github.com/wizarrrr/wizarr).

These are two entirely different projects. The correct one should be [https://github.com/wizarrrr/wizarr](https://github.com/wizarrrr/wizarr), as it doesn’t have a website, only a documentation page: [https://docs.wizarr.dev/](https://docs.wizarr.dev/).

So, I’d suggest using the GitHub repository link for the `website` field.


```suggestion
website: https://github.com/wizarrrr/wizarr
```